wiki/Client/Beatmap_editor/AiMod/en.md

```diff
@@ -25,139 +25,143 @@ Development of AiMod usually lags behind the standards held by the mapping commu
-| Letterboxing is not allowed in taiko mode. | osu!taiko maps only! |  |
+| Message | Explanation | Solution | Notes |
+| :-- | :-- | :-- |  :-- |
+| [Stack leniency](/wiki/Beatmap/Stack_leniency) is larger than 0.9 or smaller than 0.3. | The `Stack Leniency` slider in the Advanced tab of [Song Setup](/wiki/Client/Beatmap_editor/Song_Setup) is set to either less than 3 or greater than 9. | Set the `Stack Leniency` slider so that it is between 3 and 9. | This warning can safely be ignored if the usage of the stack leniency in question is intended. |
+| <!-- editor removes any unicode automatically, but this warning probably still exists -->Romanised artist contains unicode. | The `Romanised Artist` field in the General tab of [Song Setup](/wiki/Client/Beatmap_editor/Song_Setup) contains [non-standard Unicode characters](https://en.wikipedia.org/wiki/List_of_Unicode_characters). | Romanise the name of the artist in the `Romanised Artist` field following the [ranking criteria's metadata standardisation rules](/wiki/Ranking_Criteria#metadata). | |
+| <!-- editor removes any unicode automatically, but this warning probably still exists -->Romanised title contains unicode. | The `Romanised Title` field in the General tab of [Song Setup](/wiki/Client/Beatmap_editor/Song_Setup) contains [non-standard Unicode characters](https://en.wikipedia.org/wiki/List_of_Unicode_characters). | Romanise the name of the song title in the `Romanised Title` field following the [ranking criteria's metadata standardisation rules](/wiki/Ranking_Criteria#metadata). | |
+| Countdown is not allowed in mania mode. | *Self-explanatory* | Uncheck the `Enable countdown` option in the Design tab of [Song Setup](/wiki/Client/Beatmap_editor/Song_Setup). | ![](/wiki/shared/mode/mania.png) osu!mania maps only |
+| Letterboxing is not allowed in mania mode. | *Self-explanatory* | Uncheck the `Letterbox during breaks` option in the Design tab of [Song Setup](/wiki/Client/Beatmap_editor/Song_Setup). | ![](/wiki/shared/mode/mania.png) osu!mania maps only |
+| Countdown is not allowed in taiko mode. | *Self-explanatory* | Uncheck the `Enable countdown` option in the Design tab of [Song Setup](/wiki/Client/Beatmap_editor/Song_Setup). | [](/wiki/shared/mode/taiko.png) osu!taiko maps only. Although this warning exists in-game, AiMod never displays the warning as osu!taiko does not support countdown timers. |
+| Epilepsy warning is not allowed in taiko mode. | *Self-explanatory* | Uncheck the `Display epilepsy warning (storyboard has quick strobing)` option in the Design tab of [Song Setup](/wiki/Client/Beatmap_editor/Song_Setup). | [](/wiki/shared/mode/taiko.png) osu!taiko maps only. This warning is based on an archaic ranking criteria rule which has since rendered obsolete. Thus, it can safely be ignored as the current [osu!taiko ranking criteria](/wiki/Ranking_Criteria/osu!taiko) mandates that epilepsy warning is necessary for maps with strobing videos or storyboards. |
```

Comment:
```suggestion
| Epilepsy warning is not allowed in taiko mode. | *Self-explanatory* | Uncheck the `Display epilepsy warning (storyboard has quick strobing)` option in the Design tab of [Song Setup](/wiki/Client/Beatmap_editor/Song_Setup). | [](/wiki/shared/mode/taiko.png) osu!taiko maps only. This warning is based on an archaic ranking criteria rule which has since rendered obsolete. Thus, it can safely be ignored as the current [ranking criteria](/wiki/Ranking_Criteria) mandates that epilepsy warning is necessary for maps with strobing videos or storyboards. |
```
not a taiko thing, just general rc